doc/api/worker_threads.md

```diff
@@ -1958,19 +1975,16 @@ this matches its values.
 
 If the worker has stopped, the return value is an empty object.
 
-### `worker.startCpuProfile(name)`
+### `worker.startCpuProfile()`
 
 <!-- YAML
 added: REPLACEME
 -->
 
-* name: {string}
 * Returns: {Promise}
 
-Starting a CPU profile with the given `name`, then return a Promise that fulfills
-with an error or an object which has a `stop` method. Calling the `stop` method will
-stop collecting the profile, then return a Promise that fulfills with an error or the
-profile data.
+Starting a CPU profile then return a Promise that fulfills with an error
+or an [`CPUProfileHandle`][] object.
```

Comment:
If you add this type to `doc/type-parser.mjs` then you can more easily reference this here and elsewhere as just `{CPUProfileHandle}`